docker/icra2023_tutorial/Dockerfile

```diff
@@ -147,5 +147,15 @@ RUN cd /home/$USERNAME && git clone https://github.com/ros2/ros2_documentation &
 
 RUN mkdir -p /home/$USERNAME/ws_moveit2/src && cd /home/$USERNAME/ws_moveit2/src && git clone https://github.com/ros-planning/moveit2_tutorials -b humble --depth 1 && git clone https://github.com/ros-planning/moveit_task_constructor.git -b humble --depth 1
 
+# Copy Gazebo Fuel data from host if it exists
+# Assumes Dockerfile is being built from a machine that has downloaded
+# Fuel data to a directory ../fuel/ . Otherwise, no fuel data is included.
+RUN mkdir -p "/home/$USERNAME/.gz/fuel"
+# Note the weird spelling of "fuel" as the source; this is to only copy
+# if the "fuel" directory exists (via Docker glob support).  See
+# https://stackoverflow.com/questions/70096208/dockerfile-copy-folder-if-it-exists-conditional-copy/70096420#70096420
+COPY --chown=$USERNAME build.bash fue[l] /home/$USERNAME/.gz/fuel
```

Comment:
I saw the trick about a file that has to exist (build.bash).
What does the `[l]` do?

Reply:
Yep. I saw the part about "`COPY` must always have at least one valid `src` argument" when I first put in the lines, and I couldn't think of a file that's guaranteed to exist. So I abandoned that. Thanks for finding a way!